### PR TITLE
Feature: Add Field Effect Broker Support

### PR DESCRIPTION
**Pull request by FieldEffect-OperationalDevelopment**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Description
- Added support for the Field Effect Broker, allowing Endpoint and Cloud actions to be taken.

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

**Comment by FieldEffect-OperationalDevelopment** [comment]
Hello @merit-maita ; I've made this PR ready for review.  However the Review Release Notes check is failing, and I'm unsure why.  Care to guide me on how to resolve this?  Locally I've run `demisto-sdk doc-review -i Packs/CovalenceManagedSecurity/ReleaseNotes/1_2_0.md` and that passes, so I'm not sure what the error is in this step.

File: Packs/CovalenceManagedSecurity/ReleaseNotes/1_2_0.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+#### Integrations
+
+##### Covalence Managed Security
+
+- Added support for the Field Effect Broker, allowing Endpoint and Cloud actions to be taken. Adding the following commands:
+  - ***cov-mgsec-broker-ping***
+  - ***cov-mgsec-broker-list-org***
+  - ***cov-mgsec-broker-endpoint-action-by-host***
+  - ***cov-mgsec-broker-endpoint-action-by-aro***
+  - ***cov-mgsec-broker-cloud-action-by-aro***
```
Comment (merit-maita) [inline Packs/CovalenceManagedSecurity/ReleaseNotes/1_2_0.md:11]:
```suggestion
- Added support for the Field Effect Broker, allowing Endpoint and Cloud actions to be taken.
- Added 5 commands:
  - ***cov-mgsec-broker-ping***
  - ***cov-mgsec-broker-list-org***
  - ***cov-mgsec-broker-endpoint-action-by-host***
  - ***cov-mgsec-broker-endpoint-action-by-aro***
  - ***cov-mgsec-broker-cloud-action-by-aro***
```

**Review by merit-maita — COMMENTED** [review]
thank you for your contribution!
please see my comments, nothing major. and why did you make the changes to the package-lock file?

File: Packs/CovalenceManagedSecurity/Integrations/CovalenceManagedSecurity/CovalenceManagedSecurity.py [hunk 2]
```diff
@@ -313,24 +452,94 @@ def transition_aro_command():
...
+
+
+def list_organizations_broker_command(broker_instance: BrokerClient):
+    result = broker_instance.organizations()
+    return CommandResults(
+        readable_output=tableToMarkdown('Organizations', result),
+        outputs_prefix='FESBroker.Org',
+        outputs_key_field='ID',
+        outputs=result
+    )
```
[the remark this answers is not in the document]
Comment (FieldEffect-OperationalDevelopment) [reply Packs/CovalenceManagedSecurity/Integrations/CovalenceManagedSecurity/CovalenceManagedSecurity.py]: Done, thank you.

File: Packs/CovalenceManagedSecurity/Integrations/CovalenceManagedSecurity/CovalenceManagedSecurity.py [hunk 3]
```diff
@@ -374,12 +583,31 @@ def main():
...
+        elif command == 'cov-mgsec-broker-endpoint-action-by-host':
+            return_results(endpoint_action_by_host_broker_command(broker_instance, args))
+        elif command == 'cov-mgsec-broker-endpoint-action-by-aro':
+            return_results(endpoint_action_by_aro_broker_command(broker_instance, args))
+        elif command == 'cov-mgsec-broker-cloud-action-by-aro':
+            return_results(cloud_action_by_aro_broker_command(broker_instance, args))
         else:
-            msg = f'Unknown command {demisto.command()}'
+            msg = f'Unknown command {command}'
             demisto.error(msg)
```
[the remark this answers is not in the document]
Comment (FieldEffect-OperationalDevelopment) [reply Packs/CovalenceManagedSecurity/Integrations/CovalenceManagedSecurity/CovalenceManagedSecurity.py]: Done, thank you.

File: Packs/CovalenceManagedSecurity/Integrations/CovalenceManagedSecurity/CovalenceManagedSecurity.py [hunk 4]
```diff
@@ -313,24 +452,94 @@ def transition_aro_command():
...
-        if demisto.command() == 'test-module':
-            if portal_check():
+        if command == 'test-module':
+            portal_result = portal_check()
+            broker_result = bool(broker_instance.ping() == "pong")
+            if portal_result is True and broker_result is True:
                 return_results('ok')
             else:
-                return_results('nok')
+                return_results(f'Portal Check:{portal_result}, Broker Check:{broker_result}')
```
[the remark this answers is not in the document]
Comment (FieldEffect-OperationalDevelopment) [reply Packs/CovalenceManagedSecurity/Integrations/CovalenceManagedSecurity/CovalenceManagedSecurity.py]: Done, thank you.

**Review by merit-maita — COMMENTED** [review]
@FieldEffect-OperationalDevelopment let's set up a demo please

**Review by merit-maita — APPROVED** [review]

**Pull request merged**